src/files/index.ts

```diff
@@ -38,8 +38,10 @@ export const indexFileData = async (
 
   const esDocuments = Object.entries(preExistingDonorHits).map(
     ([donorId, esHit]) => {
-      const newRdpcInfo = donorFileInfo[donorId];
-      return convertToEsDocument(esHit._source, newRdpcInfo);
```

Comment:
because `newRdpcInfo` only has a new field `releaseStatus` and empty rdpc info fileds, existing es rdpc info was overwritten by `newRdpcInfo`, which resulted in empty es data in the index.